CheckDrive.Web/CheckDrive.Web/Models/Dispatcher.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace CheckDrive.Web.Models
+{
+    public class Dispatcher
+    {
+        public int Id { get; set; }
+        public int AccountId { get; set; }
+        public Account? Account { get; set; }
```

Comment:
Marking account as nullable means that we can create a Dispatcher without an account, but AccountId is required which means we cannot create an Dispatcher without AccountId.